src/server/index.jsx

```diff
@@ -206,9 +206,13 @@ server.get(
       } = getRouteProps(urlPath);
 
       const { page, renderer_env } = query;
-
+      const isOptimo = urlPath.match(/(c[a-zA-Z0-9]{10,}o)/) > 0;
```

Comment:
Do we need to check the pageType? The `isCaf` flag should be isolated enough that it can just be enabled when we're happy for the service to use it and not affect other page types.

Reply:
is this something we could move to the BFF? @amoore108

Reply:
Ohhhh, I see the pageType is set to MEDIA_ASSET anyhow.

Reply:
Not sure i know what you mean @shayneahchoon ?

Reply:
like we just have to check if pageType = 'cpsAsset'  ?

Reply:
We could, but I don't think the pagetype check is needed at all tbh? The `isCaf` flag is only used in areas where we want to apply CAF logic, so when if its enabled just for services regardless of page type, that should be ok?

Reply:
without the pageType check then this url : http://localhost:7080/pidgin/articles/cwl08rd38l6o.amp was 404 ing

Reply:
Basically, we've had some Optimo pages 404 because both Optimo articles and CAF CPS articles go through this same logic and subsequently set isCAF to true regardless of those differing pageTypes, which impacts [this url logic](https://github.com/bbc/simorgh/blob/latest/src/app/routes/utils/constructPageFetchUrl/index.ts#L69) and creates an erroneous fetch url.

Reply:
Ah ok. I've actually been looking at the area this afternoon and it may be solved with https://github.com/bbc/simorgh/pull/11518

☝️ This will check the asset ID, rather than the `isCaf` flag. We'll need to do this anyway once we drop the `isCaf` flag, but this should stabilise it a bit.

Reply:
The changes in this PR seem to work ok with the ones I have. I tested without the `pageType` as well and it seems to work ok on that AMP example page.

Reply:
Thanks!

Reply:
ah okay thank you ! will delete the pagetype check

Reply:
Good spot though 👍 Coincidence I was looking that bit 😂